packages/components/addon/components/hds/code-block/index.js

```diff
@@ -12,21 +12,19 @@ import Prism from 'prismjs';
 import { setup } from 'prismjs-glimmer';
 import { guidFor } from '@ember/object/internals';
 
-import 'prismjs/plugins/line-numbers/prism-line-numbers';
-import 'prismjs/plugins/line-highlight/prism-line-highlight';
+import { getOwnConfig, importSync } from '@embroider/macros';
```

Comment:
[Docs](https://github.com/embroider-build/embroider/blob/main/packages/macros/README.md#getownconfig-getconfig-and-getglobalconfig)